external/dynamo/.env.example

```
@@ -24,7 +24,7 @@ export DYNAMO_REPO_DIR="/path/to/NeMo-Agent-Toolkit"
 
 # GPU device IDs for unified mode (comma-separated)
 # Default: 0,1,2,3
-# DYNAMO_GPU_DEVICES="0,1,2,3"
+DYNAMO_GPU_DEVICES="0,1,2,3"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)